backend/btrixcloud/models.py

```diff
@@ -221,9 +221,59 @@ def get_raw_config(self):
 
 
 # ============================================================================
-class CrawlConfigOut(CrawlConfig):
+class CrawlConfigOut(BaseMongoModel):
     """Crawl Config Output"""
 
+    config: Optional[RawCrawlConfig]
```

Comment:
Is this mostly to make RawCrawlConfig optional here? If so, I think can still extend CrawlConfig and add `config: Optional[RawCrawlConfig]` here. If we can avoid duplicating all the other fields here..

Reply:
It is and that's what I did originally but `mypy` wasn't happy with that

Reply:
Hm, maybe there's a way to make it happy.. can take a look as well.

Reply:
I think can make it optional in CrawlConfigCore, and not optional in subclasses that require it.

Reply:
This was the complaint:
```
Found 2 errors in 1 file (checked 41 source files)
btrixcloud/models.py:232: error: Name "config" already defined on line 227  [no-redef]
...
```

Reply:
Fixed! Added a new shared `CrawlConfigAdditional` model and set config to Optional in `CrawlConfigCore`